net/https-dns-proxy/patches/030-fix-time_t-logging.patch

```diff
@@ -0,0 +1,13 @@
+--- a/src/logging.c
```

Comment:
This patch does not seems to be present in the upstream.

The proper solution for this is to backport this patch as it is:
https://github.com/aarond10/https_dns_proxy/commit/a5c4428ef963a74b8a142a6f9f3a77cf6ee0bdc5

Reply:
The upstream developed their solution based on findings/patch suggested by @schmaller. 

@schmaller given that your patch was improved on by the original developer, you can (if you want) modify this PR to include their patch or just submit the PR for the newer https-dns-proxy version from [this commit](https://github.com/aarond10/https_dns_proxy/commit/a5c4428ef963a74b8a142a6f9f3a77cf6ee0bdc5). Or just use the version you've built until I update the OpenWrt repo with the newer upstream version.

Reply:
since upstream has been fixed there's no need to patch the package anymore. right?

Reply:
@schmaller as far as I understood @aarond10 , this PR was superceded by https://github.com/openwrt/packages/pull/21177